src/providers/data_provider_fo.c

```diff
@@ -529,6 +532,18 @@ struct tevent_req *be_resolve_server_send(TALLOC_CTX *memctx,
     state->attempts = 0;
     state->first_try = first_try;
 
+    if (retry_same_server == true &&
+        state->svc->current_resolved_attempts <= MAX_CURRENT_RESOLVED_ATTEMPTS) {
+        DEBUG(SSSDBG_OP_FAILURE, "Performing retry\n");
```

Comment:
I would use `SSSDBG_IMPORTANT_INFO`
The same effect but more correct semantically.

Reply:
I would also suggest to print details (ip:port) of server being retried.

Reply:
And ideally we could log if `retry_same_server` but 2nd condition isn't met (i.e. max amount of attempts reached)

Reply:
I don't mind changing it but can you explain the difference? Currently I see that `SSSDBG_IMPORTANT_INFO` is defined as `SSSDBG_OP_FAILURE`. From what I gather they will differ in the future so what's their meaning?

Reply:
As names suggest: INFO vs FAILURE